RecoParticleFlow/PFProducer/src/PFAlgo.cc

```diff
@@ -2311,8 +2315,9 @@ void PFAlgo::createCandidatesHCAL(const reco::PFBlock& block,
           particleDirection.push_back(std::get<1>(pae));
           ecalEnergy.push_back(mergedPhotonEnergy * clusterEnergyCalibrated / sumEcalClusters);
           hcalEnergy.push_back(0.);
-          rawecalEnergy.push_back(totalEcal);
-          rawhcalEnergy.push_back(totalHcal);
+          rawecalEnergy.push_back(mergedPhotonEnergy * clusterEnergyCalibrated /
+                                  sumEcalClusters);  // "raw" not well defined. use the corrected value.
+          rawhcalEnergy.push_back(0.);
```

Comment:
this breaks the HO energy settings .

Some protections are necessary below in 
https://github.com/cms-sw/cmssw/blob/18425f38932959335ace0f686b663eacfa129e69/RecoParticleFlow/PFProducer/src/PFAlgo.cc#L2374-L2377

Reply:
Thank you @slava77 . I made the change.